configure.ac

```diff
@@ -283,11 +283,19 @@ case "${host}" in
 	abi="elf"
 	AC_DEFINE([JEMALLOC_PURGE_MADVISE_FREE], [ ])
 	;;
-  *-*-openbsd*|*-*-bitrig*)
+  *-*-openbsd*)
 	CFLAGS="$CFLAGS"
 	abi="elf"
 	AC_DEFINE([JEMALLOC_PURGE_MADVISE_FREE], [ ])
+	force_tls="0"
```

Comment:
@dhuseby `force_tls=0` is needed (for openbsd) for the **rust-lang/jemalloc** fork, but not for **jemalloc/jemalloc**. rustc don't use the latest jemalloc version, and so autodetection don't work very well for rustc version of jemalloc.